plugins/perceptionexpander.koplugin/main.lua

```diff
@@ -128,7 +128,7 @@ function PerceptionExpander:showSettingsDialog()
             {
                 text = "",
                 input_type = "number",
-                hint = T(_("Increase margin after pages. Current value: %1"),
+                hint = T(_("Increase margin after pages. Current value: %1\n Set to 0 to disable."),
```

Comment:
`\n Set` the space probably shouldn't be there, as it may result in text looking like this
```
Increase margin after pages. Current value: %1
 Set to 0 to disable
```
Otherwise seems fine to me